server/player.lua

```diff
@@ -835,20 +835,21 @@ function CreatePlayer(playerData, Offline)
     ---@deprecated use ox_inventory exports directly
     ---@param item string
     ---@param amount number
-    ---@param slot? number
     ---@param metadata? table
+    ---@param slot? number
     ---@return boolean success
-    function self.Functions.AddItem(item, amount, slot, metadata)
-        return exports.ox_inventory:AddItem(self.PlayerData.source, item, amount, slot, metadata)
+    function self.Functions.AddItem(item, amount, metadata, slot)
+        return exports.ox_inventory:AddItem(self.PlayerData.source, item, amount, metadata, slot)
     end
 
     ---@deprecated use ox_inventory exports directly
     ---@param item string
     ---@param amount number
+    ---@param metadata? table
     ---@param slot? number
     ---@return boolean success
-    function self.Functions.RemoveItem(item, amount, slot)
-        return exports.ox_inventory:RemoveItem(self.PlayerData.source, item, amount, nil, slot)
+    function self.Functions.RemoveItem(item, amount, metadata, slot)
+        return exports.ox_inventory:RemoveItem(self.PlayerData.source, item, amount, metadata, slot)
```

Comment:
```suggestion
    function self.Functions.RemoveItem(item, amount, slot)
        return exports.ox_inventory:RemoveItem(self.PlayerData.source, item, amount, nil, slot)
```

QBCore's implementation didn't support removal filter via metadata